pkg/cephmgr/cephd/cephd.go

```diff
@@ -90,6 +90,24 @@ func GetCephdError(err int) error {
 	return cephdError(err)
 }
 
+// run a command in libcephd
+func RunCommand(command string, args []string) error {
+
+	// The command passes through args to the child process.  Look for the
+	// terminator arg, and pass through all args after that (without a terminator arg,
+	// FlagSet.Parse prints errors for args it doesn't recognize)
+	passthruIndex := 3
```

Comment:
we may need an override some time that allows the caller to specify their known pass through index, but if everyone is using 3 for now then that doesn't matter.